tests/Unit/PrestaShopBundle/ApiPlatform/Metadata/CQRSDeleteTest.php

```diff
@@ -56,13 +56,13 @@ public function testDefaultConstructor(): void
 
         // With named parameters
         $operation = new CQRSDelete(
-            output: true,
+            output: 'test',
```

Comment:
strong typing has been added (array|string|false|null)

Reply:
Hmm, ok as long as the default value is still false I guess we're good